packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2193,6 +2193,23 @@ class ShaderMaskLayer extends ContainerLayer {
   }
 }
 
+/// A backdrop key uniquely identifies the backdrop that a [BackdropFilterLayer]
+/// samples from.
+///
+/// When multiple backdrop filters share the same key, the Flutter engine can
+/// more efficiently perform the backdrop operations.
+///
+/// For more information, see `BackdropFilter`.
+@immutable
+final class BackdropKey {
+  /// Create a new [BackdropKey].
+  BackdropKey() : _key = _nextKey++;
```

Comment:
There is example code on the BackdropFilter widget. 

> Maybe we can simply use a ValueKey so that developers can use the same literal value to indicate identity?

That would require us to traverse the layer tree each frame to convert the value keys into a serializable value, since the engine doesn't know what a dart value key is. 

> Depending on how the backdrop configuration should be shared, it might not be the easiest way to use

Can you be more specific? You should think of BackdropKey a bit like a GlobalKey. The expectation is that you either create it once in a static somewhere and cache it, or use the BackdroupGroup to automatically manage it for you.

Reply:
I see. Is it correct to say that `BackdroupGroup` is preferred over `BackdropKey`, and can you document it?

Reply:
Yeah I would say backdropGroup would be preferred. I can make sure the documentation emphaiszes this.

Reply:
I added documentation to call out the usage of BackdropGroup